pkg/universe.dagger.io/x/hardy4yooz@gmail.com/notify/discord/discord.cue

```diff
@@ -0,0 +1,35 @@
+package discord
```

Comment:
Those are changes from another PR, can you rebase your PR?

Reply:
@TomChv I tried rebasing but the DCO check keeps failing. Facing some difficulties with Git too. I'll close and open another PR.